src/ExtensionGallery/Controllers/ApiController.cs

```diff
@@ -46,12 +49,17 @@ public object Get(string id)
 		}
 
 		[HttpPost]
-		public async Task<IActionResult> Upload([FromQuery]string repo, string issuetracker)
+		public async Task<IActionResult> Upload()
 		{
+			var context = _httpContextAccessor.HttpContext;
+
+			Stream bodyStream = context.Request.Body;
+			string repo = context.Request.Query["repo"];
+			string issueTracker = context.Request.Query["issuetracker"];
```

Comment:
For some reason, .NET Core seems to ignore the request body if the querystring is being auto-parsed into arguments.  I just pulled them out manually as a workaround.